deployer/scripts/hawkular.sh

```diff
@@ -19,48 +19,23 @@
 function deploy_hawkular() {
```

Comment:
ansible file of interest that needs changing: https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_metrics/tasks/generate_hawkular_certificates.yaml